dbms/src/Server/Server.cpp

```diff
@@ -207,6 +207,9 @@ int Server::main(const std::vector<std::string> & args)
 	global_context->setGlobalContext(*global_context);
 	global_context->setApplicationType(Context::ApplicationType::SERVER);
 
+	//https init, may be used in dictionaries
+	clientSSLInit();
```

Comment:
You forgot about LocalServer.

Why not to use singleton-like initialization (either `Singleton`, or local static variable or `std::call_once`).